db/models.py

```diff
@@ -1,4 +1,10 @@
+
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
+from django.db.models import UniqueConstraint
+
+import settings
```

Comment:
it's better to use
`from django.conf import settings`
because you may want to have settings folder or another settings file